tests/llmcompressor/transformers/compression/recipes/new_quant_weight.yaml

```diff
@@ -14,4 +14,3 @@ test_stage:
                     targets: ["Linear", "Embedding"]
         GPTQModifier:
             block_size: 128
-            targets: ["re:model.layers.\\d+$"]
```

Comment:
Pretty sure this was failing because the change to use `match_named_targets` isn't applying quantization config to non-quantizable layers, so having `targets` that aren't the leaf Linear layers will fail out now. Do we want to support this target to mean "all modules within target module" as well?